historical_data_api_client/cli.py

```diff
@@ -0,0 +1,110 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+import json
+import logging
+import os
+from datetime import datetime
```

Comment:
May be declaring common imports under `historical_data_api_client/__init__.py`
https://github.com/gridsingularity/d3a-interface/pull/230/files#r664487664

Reply:
This is not the point of the mentioned discussion. The __init__ is not the place for common imports. I could add a `__all__`  variable in order to more easily access modules inside of the historical_data_api package from outside. But this will probably never be the case.
But maybe you meant something else here?

Reply:
>  I could add a __all__ variable in order to more easily access modules inside of the historical_data_api package from outside.
Yes thats what I meant

> But this will probably never be the case.

Skip it then, if you think so